### PR TITLE
Preferences

### PR DESCRIPTION
**Pull request by akinwale**

### CODE REVIEW

File: src/component/AppNavigator.js [hunk 1]
```diff
@@ -292,8 +296,17 @@ class AppWithNavigationState extends React.Component {
...
 
   componentDidMount() {
+    const { dispatch } = this.props;
     this.emailVerifyCheckInterval = setInterval(() => this.checkEmailVerification(), 5000);
     Linking.addEventListener('url', this._handleUrl);
+
+    // call /sync/get with interval
+    setInterval(() => {
+      this.setState({ syncHashChanged: false }); // reset local state
+      NativeModules.UtilityModule.getSecureValue(Constants.KEY_FIRST_RUN_PASSWORD).then(walletPassword => {
```
Comment (kauffj) [inline src/component/AppNavigator.js]: this constant should probably just be wallet password if it's safe to change

File: src/component/AppNavigator.js [hunk 2]
```diff
@@ -29,7 +29,7 @@ import {
 import { connect } from 'react-redux';
```
Comment (kauffj) [inline src/component/AppNavigator.js]: none of the logic in this file appears to affect the presentation or rendering of this component

is there a design that would separate this out more cleanly? maybe just make it it's own component?

File: src/component/AppNavigator.js [hunk 3]
```diff
@@ -292,8 +296,17 @@ class AppWithNavigationState extends React.Component {
   }
 
   componentDidMount() {
+    const { dispatch } = this.props;
     this.emailVerifyCheckInterval = setInterval(() => this.checkEmailVerification(), 5000);
     Linking.addEventListener('url', this._handleUrl);
+
+    // call /sync/get with interval
+    setInterval(() => {
```
Comment (kauffj) [inline src/component/AppNavigator.js]: you should probably store the result of this `setInterval` call and call `clearInterval` on unmount

**Comment by kauffj** [comment]
Additional commentary on this PR coming on https://github.com/lbryio/lbry-desktop/issues/3022 momentarily

**Comment by akinwale** [comment]
Closing in favour of https://github.com/lbryio/lbry-react-native/pull/57

**Pull request closed without merging**